server/models/Product.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const productSchema = new mongoose.Schema({
+    name: {
+      type: String,
+      required: true
+    },
+    description: {
+      type: String
+    },
+    url: {
+      type: String,
+      required: true
+    },
+    pictureUrl: {
+      type: String,
+      required: true
+    },
+    price: {
+      type: Number,
+      required: true
+    },
+    userId: {
```

Comment:
Hey @izzangalaxybyte, I don't think it's efficient to set the userId field in the produced documents, you can set it once in the list documents, as the list is directly linked to the user who created it.

Reply:
fixed